content/docs/command-reference/studio/token.md

```diff
@@ -0,0 +1,24 @@
+## studio token
+
+View the token dvc uses to contact Studio
+
+## Synopsis
+
+```usage
+usage: dvc studio token [-h] [-q | -v]
```

Comment:
what is the use of this command btw? it's not very secure to my mind ..

Reply:
It is similar to how we have `gh auth token`. User can use this command in CI or someplace to see the used token and use the value accordingly.